lib/adapters/http.js

```diff
@@ -664,7 +664,8 @@ function HttpPouch(opts, callback) {
 
       var keysAsString =
         'keys=' + encodeURIComponent(JSON.stringify(opts.keys));
-      if (keysAsString.length + params.length + 1 <= MAX_URL_LENGTH) {
+      // FIX for sync gateway, which does not accept multiple keys on a GET request
+      if (false) { //keysAsString.length + params.length + 1 <= MAX_URL_LENGTH) {
```

Comment:
@ssured I couldn't find anything documenting this, do you remember where does it come from please ?